src/RadixDlt.NetworkGateway.GatewayApiSdk/Model/ComplexKeysCursor.cs

```diff
@@ -67,17 +67,17 @@
 namespace RadixDlt.NetworkGateway.GatewayApiSdk.Model;
 
 [DataContract]
-public sealed record StateKeyValueStoreKeysCursor(long? StateVersionBoundary, long? IdBoundary)
+public sealed record ComplexKeysCursor(long? StateVersionBoundary, long? IdBoundary)
```

Comment:
Complex keys? If it's ID-based cursor why won't we just call it `IdBoundaryCursor`?

Reply:
The cursor for all of these entry paginations will be `(state version, entry definition id)`

Reply:
_But_ I don't think that would produce a performant query because the bound on `from_state_version` wouldn't mesh with the `entry_id` bound (because the DB engine doesn't know that `from_state_version` is increasing with `entry_id`)